app/portainer/views/devices/import/importDevice.html

```diff
@@ -99,6 +89,39 @@
               </div>
             </div>
             <!-- !device name input -->
+            <!-- suffix input -->
+            <span class="small">
+              <p class="text-muted" style="margin-top: 10px">
+                <i class="fa fa-exclamation-circle blue-icon" aria-hidden="true" style="margin-right: 2px"></i>
+                Suffix starting number will be appended to the end of the Device name, if initiating multiple devices this will be incrementally increased
+              </p>
+            </span>
+            <div class="form-group">
+              <label for="suffix" class="col-sm-3 col-lg-2 control-label text-left"> Suffix starting number </label>
+              <div class="col-sm-9 col-lg-10">
+                <input
+                  type="text"
```

Comment:
You can force the html input to be a positive integer, no need to use the `pattern` prop
```suggestion
                  type="number"
                  min="1"
```

Reply:
Yeah, that doesn't really work